src/components/SegmentedTopBar/Tab.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { forwardRef, useCallback } from "react";
+import { Pressable } from "react-native";
+
+import PropTypes from "prop-types";
+
+import { Typography } from "@components";
+
+export const Tab = forwardRef(({ label, value, navigation, ...rest }, ref) => {
+  const handleOnPress = useCallback(
+    () => navigation.navigate(value),
+    [navigation, value]
+  );
+
+  return (
+    <Pressable
```

Comment:
@chiragbhaiji We are losing ripple effect, instead we can use combination of view and touchable like below.


```
<View ref={ref} style={{ flex }}>
      <Touchable
        flex={1}
        justifyContent="center"
        onPress={handleOnPress}
        {...rest}
      >
        <Typography
          fontFamily="sf500"
          fontSize="xs"
          numberOfLines={1}
          textAlign="center"
        >
          {label}
        </Typography>
      </Touchable>
    </View>
```